crates/redeem/src/mock.rs

```diff
@@ -343,6 +351,15 @@ impl ExtBuilder {
         .assimilate_storage(&mut storage)
         .unwrap();
 
+        GenesisBuild::<Test>::assimilate_storage(
+            &loans::GenesisConfig {
+                max_exchange_rate: Rate::from_inner(DEFAULT_MAX_EXCHANGE_RATE),
+                min_exchange_rate: Rate::from_inner(DEFAULT_MIN_EXCHANGE_RATE),
+            },
```

Comment:
Wondering if we can implement `Default` for this at source to use some sensible values then we don't need to re-declare this everywhere